public/css-variables.css

```diff
@@ -0,0 +1,105 @@
+/* Dark skin. */
+.css-variables {
+
+    /*
+      Styling is only supported using the following CSS variables.
+      They will be maintained in a backwards-compatible manner as good as possible.
+      It is technically possible to add any valid CSS to this file,
+      but there are no guarantees with respect to backwards-compatibility.
+
+      Variable types:
+      - variables ending with -color: any valid CSS color
+      - variables ending with -border: any value valid for the CSS border shorthand property (e.g. 1px solid red)
+      - border-radius: any positive CSS length/percentage
+      - variables ending with -opacity: any positive CSS float value (e.g. 0.5)
+    */
+
+    /* General */
+    --background-color: #202020;
+    --text-color: #ddd;
+    --error-text-color: #dc4646;
+    --error-field-background-color: var(--error-text-color);
+    --success-text-color: #46c846;
+    --neutral-text-color: #bbb;
+    --border-radius: 5px;
+
+    /* Buttons */
+    --button-icon-color: var(--text-color);
+    --button-add-icon-color: var(--success-text-color);
+    --button-remove-icon-color: var(--error-text-color);
+    --button-text-color: var(--text-color);
+    --button-background-color: #333;
+    --button-border: 1px solid #999;
```

Comment:
--button-border: 2px solid rgb(70, 70, 70);

Reply:
The same change works on all borders, makes them a little wider and darker, and the whole thing gets a little easier on the eyes.